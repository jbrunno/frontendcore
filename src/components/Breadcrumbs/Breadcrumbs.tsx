import React from 'react';
import {
  Breadcrumbs as MuiBreadcrumbs,
  BreadcrumbsProps as MuiBreadcrumbsProps,
} from '@mui/material';

export type BreadcrumbsProps = MuiBreadcrumbsProps;

export function Breadcrumbs(props: BreadcrumbsProps) {
  return <MuiBreadcrumbs {...props} />;
}
export default Breadcrumbs;
