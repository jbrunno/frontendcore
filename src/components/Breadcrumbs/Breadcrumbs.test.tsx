import React from 'react';
import { render } from '@test-utils/component-testing';
import { Breadcrumbs } from './Breadcrumbs';

test('renders the Breadcrumbs', () => {
  const { container } = render(<Breadcrumbs />);
  expect(container).toMatchSnapshot();
});
