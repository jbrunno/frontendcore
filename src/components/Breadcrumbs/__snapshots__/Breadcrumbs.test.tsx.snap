// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Breadcrumbs 1`] = `
.emotion-0 {
  margin: 0;
  font-weight: 400;
  font-size: 1rem;
  line-height: 1.5rem;
  letter-spacing: 0.0094rem;
  font-family: Lato,sans-serif;
  color: #3B3B4199;
}

.emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0;
  margin: 0;
  list-style: none;
}

<div>
  <nav
    class="MuiTypography-root MuiTypography-body1 MuiBreadcrumbs-root emotion-0"
  >
    <ol
      class="MuiBreadcrumbs-ol emotion-1"
    />
  </nav>
</div>
`;
